### PR TITLE
fixed db driver import problem of adm.

### PR DESCRIPTION
**Pull request by JasonHonor**
Fixed a problem caused by version 1.0.5.

### CODE REVIEW

File: plugins/admin/modules/helper.go [hunk 1]
```diff
@@ -39,7 +40,8 @@ func RemoveBlankFromArray(s []string) []string {
 }
 
 func Uuid() string {
-	return uuid.NewV4().String()
```
Comment (chenhg5) [inline plugins/admin/modules/helper.go]: See the latest release version of package go.uuid, https://github.com/satori/go.uuid/blob/v1.2.0/generator.go#L64.

**Comment by chenhg5** [comment]
@JasonHonor Thanks for pr. Can you check the commit of go.uuid and make it work? And then i will merge it.

File: plugins/admin/modules/helper.go (hunk 1 above)
Comment (JasonHonor) [reply plugins/admin/modules/helper.go]: It work fine when I update the package go.uuid, thanks.

Comment (JasonHonor) [reply plugins/admin/modules/helper.go]: #77

**Pull request merged**